src/sass/variables/_Animation.Variables.MDL2.scss

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
```

Comment:
I know it's implied by the filename, but we should add a comment in the file that these are now deprecated

Reply:
Added